transport.go

```diff
@@ -284,6 +285,14 @@ func getRequestFromEvent(event *Event, dsn *Dsn) (r *http.Request, err error) {
 	if err != nil {
 		return nil, err
 	}
+	if ctx != nil {
+		return http.NewRequestWithContext(
+			ctx,
+			http.MethodPost,
+			dsn.GetAPIURL().String(),
+			envelope,
+		)
+	}
 	return http.NewRequest(
```

Comment:
You don't need to leave this here, just set `ctx = context.Background()` if it's `nil` and call `NewRequestWithContext`.